### DISCUSSION TITLE
Weird horizontal lines after BL Touch install & update to BL Touch Firmware.

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#393] [Q&A] [answered]
Image: 
![image](https://user-images.githubusercontent.com/15057335/113710296-3294e880-96ec-11eb-812c-005fd93d4014.png)

As you can see there's lines, they appear always on same layers as well, on all prints. Prior this update everything worked smoothly.

Thing's I've checked:
- X/Y belt tension.
- Concentric Nuts of both bed & gantry.
- Z Axis checked if it was binding, not the case.
- Z axis reapplied lubricant to lead screw.
- Frame is square and straight.

These are the things I've checked to make sure it's not the case. Prior BL Tocuh firmware I was on Default version, which worked flawlessly. After it became very visible that there's some sort of layer shifts.

**Comment by Jyers** [comment]
You could try disabling the mesh in your start gcode and see if the lines persist. That would give some indication as the whether this is actually caused by the BL Touch

**Reply by VashBaldeus** [reply]
> 
> 
> You could try disabling the mesh in your start gcode and see if the lines persist. That would give some indication as the whether this is actually caused by the BL Touch

Tried without, lines still there, I am running out of ideas what to checked to try figure out why it is happening.

**Reply by Jyers** [reply]
The only thing I can think is that you knocked something while installing the BL Touch.  You could also try a NoProbe firmware just to rule out the firmware as an issue.

**Reply by G4GNX** [reply]
I wonder if this is an electromechanical issue, rather than BLTouch or firmware?
It's a long shot, but when you installed the BLTouch, you may have disturbed the frame/case grounding, simply by moving the screws to get the cover off.
Creality have made a big fuss and a horrible video to show how to cure grounding issues causing layer shifting.
It could afford you a fix by following the principles in the video, just not the method, as you will see.
I don't have layer shifting, but I'm going to perform the grounding procedure anyway, because I'm OCD about RFI.

https://www.youtube.com/watch?v=U_eVpsNJd8A

**Comment by Jyers** [comment, marked as answer]
Did you ever find a cause for this issue?

**Reply by VashBaldeus** [reply]
Well, it is definitely not the Z Axis binding, and I managed to reduce those lines almost to non-existent. Those lines were there when I got the printer as well, but less noticeable, I managed to reduce those to what I had at the beginning.

**Comment by smooreace** [comment]
Thats friggin HILARIOUS!

**Reply by G4GNX** [reply]
What is? The video? Made by a butcher, but it shows what's needed.
